charts/generic-service/values.yaml

```diff
@@ -209,6 +209,7 @@ devopsToolsImage: &devopsToolsImage ghcr.io/ministryofjustice/hmpps-devops-tools
 devopsToolsTag: &devopsToolsTag main
 
 postgresDatabaseRestore:
+  jobName:
```

Comment:
Maybe a comment would be useful here (much as I dislike them) to explain why this property is blank?

Ultimately the problem as I understand it is that the name cannot be > 64 characters.

If the name is arbitrary and just needs to be human-readable, could it simply be truncated rather than overridden?